### DISCUSSION TITLE
Edit experience

### DISCUSSION THREAD

**Discussion by char-mainewang** [twilio-labs/paste#2973] [Office hours] [answered]
Context: RETL destinations are built on top the event streaming Actions destination framework where customers can create different mappings to configure how they want their data to be sent to the destination. Event streaming destinations have a mapping table where users can find all the controls (i.e. enable/disable, edit, delete). However, each RETL mappings have its own page that customer navigate to through clicking on the a row. Most folks land on the mapping page as their last step and often want to edit or view their mapping setup.

Goal: We'd love to make those controls accessible on the mapping page so that customers don't have to navigate back to the mapping table.

Feedback I'm looking for:
- where the controls should be placed 
- when should the editing experience be behind an edit button vs in a tab like settings

**Comment by loreina** [comment, marked as answer]
Marking as answered as the discussion was resolved in office hours. Thanks!
